src/Components/OATPropertyEditor/Internal/PropertyList/Internal/PropertyListItem/Internal/PropertyIcon/PropertyIcon.tsx

```diff
@@ -38,6 +38,13 @@ const PropertyIcon: React.FC<IPropertyIconProps> = (props) => {
 
     // data
     const { iconSource, iconTitle, iconType } = useMemo(() => {
+        if (overrideIcon) {
+            return {
+                iconSource: overrideIcon.name,
+                iconTitle: overrideIcon.title,
+                iconType: 'Fluent'
```

Comment:
why not overrideIconType?

Reply:
cause today it will always be an icon by name